DataSetExplorer/ConsoleApp/ConsoleIO.cs

```diff
@@ -0,0 +1,55 @@
+﻿using DataSetExplorer.DataSetBuilder.Model;
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.IO;
+
+namespace DataSetExplorer.ConsoleApp
+{
+    internal class ConsoleIO
+    {
+        internal static void ChosenOptionErrorMessage(string chosenOption)
+        {
+            Console.WriteLine("Option " + chosenOption + " not found. Choose again.\n");
+        }
+
+        internal static string GetAnswerOnQuestion(string question)
+        {
+            Console.Write(question);
+            return Console.ReadLine();
+        }
+
+        internal static ListDictionary GetAnnotatedProjects()
```

Comment:
This and the method below seem misplaced. Is there a better location to store them? If not leave them here, but add a comment that they should be moved somewhere, as they are not "generic enough" to belong to the broad ConsoleIO utility class.